packages/react-examples/src/react-button/MenuButton/MenuButton.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { MenuButton, MenuButtonProps } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuProps, MenuTrigger } from '@fluentui/react-menu';
+import { buttonBaseProps } from '../Button/Button.stories';
+import { Playground } from '../Playground';
+import { PlaygroundProps } from '../Playground.types';
+
+const ExampleMenu = (props: MenuButtonProps): JSX.Element => {
+  const { expanded, onMenuDismiss } = props;
+
+  const onOpenChange: MenuProps['onOpenChange'] = (ev, data) => {
+    if (!data.open) {
+      onMenuDismiss?.();
+    }
+  };
+
+  return (
+    <Menu open={expanded} onOpenChange={onOpenChange}>
+      <MenuTrigger>
+        <MenuButton {...props} />
+      </MenuTrigger>
```

Comment:
I would suggest to remove this, the DOM attribute spreading should work right now

```suggestion
  return (
    <Menu>
      <MenuTrigger>
        <MenuButton/>
      </MenuTrigger>
```

Reply:
Simplified this story a lot.